sql_csv_utils.py

```diff
@@ -104,6 +104,45 @@ def check_agent_name_sql(self, first_name: str, last_name: str, middle_initial:
         else:
             return result
 
+
```

Comment:
new function for telling if a hybrid is already in database, so SpeciesA x SpeciesB will be recognized as the same hybrid as Species B x Species A, so it looks for the presence of all substrings instead of an exact match. will defer to standard database search in a hybrid with a single word name "x ambigua"